### ISSUE TITLE
P2PP  generates error

### ISSUE THREAD

**Issue by cinealfa** [tomvandeneede/p2pp#188]
<img width="918" alt="Screenshot 2024-01-21 at 15 28 14" src="https://github.com/tomvandeneede/p2pp/assets/32920432/07570a0d-3473-452f-b825-bbdc03eaf619">
I am getting this error what am I doing wrong?

**Comment by tomvandeneede** [comment]
can you please send me a copy of the file shown there ( /var/…..) and I will have a lookOp 22 jan 2024 om 00:32 heeft cinealfa ***@***.***> het volgende geschreven:﻿
Screenshot.2024-01-21.at.15.28.14.png (view on web)
I am getting this error what am I doing wrong?

**Comment by cinealfa** [comment]
Here you are!
[PrusaSlicer.upload.dfac-4a5d-8a7f-63c2.txt](https://github.com/tomvandeneede/p2pp/files/14032786/PrusaSlicer.upload.dfac-4a5d-8a7f-63c2.txt)

**Comment by cinealfa** [comment]
> can you please send me a copy of the file shown there ( /var/…..) and I will have a lookOp 22 jan 2024 om 00:32 heeft cinealfa ***@***.***> het volgende geschreven:﻿ Screenshot.2024-01-21.at.15.28.14.png (view on web) I am getting this error what am I doing wrong? —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

**Comment by cinealfa** [comment]
Any thoughts?

**Comment by tomvandeneede** [comment]
looks like the file is not in utf-8 formatwhat version of ps are you using?can you send me the filementioned in the screenshot (txt file)Op 27 jan 2024 om 02:03 heeft cinealfa ***@***.***> het volgende geschreven:﻿

can you please send me a copy of the file shown there ( /var/…..) and I will have a lookOp 22 jan 2024 om 00:32 heeft cinealfa @.> het volgende geschreven:﻿ Screenshot.2024-01-21.at.15.28.14.png (view on web) I am getting this error what am I doing wrong? —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you are subscribed to this thread.Message ID: @.>

**Comment by cinealfa** [comment]
i did it is in the message right after you asked me to send it, Did I misunderstand you?

**Comment by tomvandeneede** [comment]
yep got that but that was not a code file at all so this is definitely something P2PP cannot work with.
Which version of Prusa Slicer are you using?   Maybe there is a new file format that gets used now that P2PP is not (yet) aware of.

￼

Would be interesting to understand if this is the file BEFORE processing or AFTER
if after processing has been going on it would definitely be good to get a copy of the file BEFORE the processing by P2PP

**Comment by tomvandeneede** [comment]
Looking at the actual error it appears to be happening in the reading of the file

￼

so the document you send is the input file from purse….

Can you please let me know what you have under printer settings/general and then the code flavour a thumbnail settings?  
I am expecting all data to be text encoded but in your file this seems no longer the case

￼

**Comment by cinealfa** [comment]
Thank you for the assistance,  I m using 2.7.1 for OSX I will get to that computer and output the settings and check the flavor (which I believe is Klipper) and report back.

**Comment by tomvandeneede** [comment]
Don't believe this has anything to do with the MACOs.   I also use MAC and have no issues.  The main problem seems to be the output file is not an uft-8 based text file but contains other data and P2PP is not able to process it

**Comment by cinealfa** [comment]
What would be causing this? I I swear I followed the instructions in the PDF to the letter. Should I uninstall and start over? I haven't been tinkering under the hood with any of the 3-D printing stuff I use so I'm a little bit at a loss what I'm doing wrong.

**Comment by sorgens** [comment]
I’ve just noticed that turning off ‘binary G-code’ resolves the situation.
